user-guide/Advanced_Functionality/Swarming/Swarming.md

```diff
@@ -4,80 +4,43 @@ uid: Swarming
-
-## Capabilities
-
-The Swarming feature provides these capabilities:
+Swarming can be useful in many situations:
 
-- As a DataMiner System Admin, you can apply maintenance (e.g. Windows updates) on a live cluster, Agent by Agent, by temporarily moving functionalities away to other Agents in the cluster.
+- As a System Administrator, you can **eliminate downtime** resulting from maintenance activities for a more polished user experience, by carrying out the maintenance activities (e.g. Windows updates) on a live cluster, Agent by Agent, by temporarily moving functionalities away to other Agents in the cluster.
 
-- As a DataMiner System Admin, you can easily extend your system with an extra node and move functionalities from existing nodes to new nodes, so you can rebalance your cluster.
+  ![Maintenance without downtime](~/user-guide/images/Swarming_maintenance.png)
 
-- Swarming makes it possible to recover functionalities from failing nodes by moving activities hosted on such a node to the remaining nodes.
+- With swarming, **load balancing** is easy. You can easily move elements between nodes to rebalance your cluster without effort, for example when you have added an extra node or when you want to reduce the load on a specific node.
 
-In a later iteration, the Swarming feature will also be able to assist in making rolling DataMiner software updates on live clusters possible, with limited downtime of specific functionality.
+  ![Load balancing using Swarming](~/user-guide/images/Swarming_load_balance.png)
 
-> [!NOTE]
-> The above capabilities are possible with limited downtime and as long as there is spare capacity.
+- If a node in a cluster fails, you can quickly recover functionalities from the failing node by moving activities hosted on that node to the remaining nodes.
 
-## Limitations
+- In a later iteration, the Swarming feature will also be able to assist in making rolling DataMiner software updates on live clusters possible, with limited downtime of specific functionality.
 
-Some functionality is not supported with the Swarming feature. These are the most important limitations at the moment:
+For all of these capabilities, it is important that the different nodes in a cluster have spare capacity so that elements can be moved to them. For optimal ease of use of the Swarming feature, we therefore recommend deploying DataMiner in [subscription mode](xref:Pricing_Commercial_Models) and using as many DataMiner nodes as necessary to make sure there is plenty of capacity to move elements whenever needed.
 
-- Swarming is not available in DataMiner Systems with a [storage per DMA setup](xref:Configuring_storage_per_DMA) (Cassandra or MySQL).
+Swarming can be enabled from DataMiner 10.5.1/10.6.0 onwards<!-- RN 41490 -->. Prior to this, this feature is available in preview if the *Swarming* [soft-launch option](xref:SoftLaunchOptions) is enabled.
```

Comment:
do we want to advertise the softlaunch?

Reply:
Should users not use the soft-launch option? If that is the case, I'd better remove the information about it completely.